lib/vanagon/platform/dsl.rb

```diff
@@ -44,7 +44,7 @@ def platform(platform_name, &block)
                       Vanagon::Platform::Solaris10.new(@name)
                     when /^solaris-11/
                       Vanagon::Platform::Solaris11.new(@name)
-                    when /^windows-/
+                    when /^(windows|windowsfips)-/
```

Comment:
what if this was just `/^windows/`?

Reply:
I used the same approach as for redhatfips :) Not sure about the implications of removing the `-`